composer.json

```diff
@@ -16,13 +16,17 @@
         }
     },
     "require": {
-        "codeception/base": "^2.5 || ^3.0.0",
-        "composer/semver": "^1.4"
+        "php": "^7.1",
+        "ext-json": "*",
+        "codeception/codeception": "^4.0.3",
+        "composer/semver": "^1.4",
+        "codeception/module-cli": "^1.0.0",
+        "codeception/module-filesystem": "^1.0.2"
     },
     "require-dev": {
         "squizlabs/php_codesniffer": "^3.4",
-        "vimeo/psalm": "^3.0",
-        "phpunit/phpunit": "^6.0.0 || ^7.0.0 || ^8.0.0" 
+        "vimeo/psalm": "^3.7.0",
```

Comment:
What's the point of not requiring the latest version of a *dev* dependency ? We won't have have the latest bugfixes and static analysis report might be different since we are doing it for every combination of the matrix.

Reply:
The module acts as a driver for the psalm cli, so it needs to be tested for compatibility with different psalm versions. Running static analysis on the module itself is secondary to that.

In fact psalm is a runtime dependency for this module. I can't recall why I decided to declare it as a dev-dep, but I'm fairly sure I had a reason for that.

Reply:
> In fact psalm is a runtime dependency for this module. I can't recall why I decided to declare it as a dev-dep, but I'm fairly sure I had a reason for that.

To be able to let users chose the PHAR version of Psalm instead :)

Reply:
That could be it. Psalm plugins often break when used with Psalm phars though, due to the random namespace scoping Psalm uses for phar builds.